interface/main/finder/dynamic_finder.php

```diff
@@ -286,7 +286,7 @@
         if (!empty($GLOBALS['gbl_pt_list_new_window'])) {
             echo ' checked';
         }
-        ?> /><?php echo xlt('Open in New Window'); ?></label><label for='setting_search_type' id='setting_search_type_label'><input type='checkbox' name='setting_search_type'  id='setting_search_type' onchange='persistCriteria(this, event)' value='<?php echo attr($patient_finder_exact_search); ?>'<?php echo text($patient_finder_exact_search); ?>/><?php echo xlt('Search with exact method'); ?></label></form>");
+        ?> /><?php echo xlt('Open in New Window'); ?></label>&nbsp;<label for='setting_search_type' id='setting_search_type_label'><input type='checkbox' name='setting_search_type'  id='setting_search_type' onchange='persistCriteria(this, event)' value='<?php echo attr($patient_finder_exact_search); ?>'<?php echo text($patient_finder_exact_search); ?>/><?php echo xlt('Search with exact method'); ?></label></form>");
```

Comment:
can we just use margin classes instead of `&nbsp`. 
`
$("div.mytopdiv").html("<form name='myform'><label for='form_new_window' id='form_new_window_label'><input class='mx-1' type='checkbox' id='form_new_window' name='form_new_window' value='1'<?php
        if (!empty($GLOBALS['gbl_pt_list_new_window'])) {
            echo ' checked';
        }
        ?> /><?php echo xlt('Open in New Window'); ?></label><label for='setting_search_type' id='setting_search_type_label'><input type='checkbox' name='setting_search_type' class='mx-1'  id='setting_search_type' onchange='persistCriteria(this, event)' value='<?php echo attr($patient_finder_exact_search); ?>'<?php echo text($patient_finder_exact_search); ?>/><?php echo xlt('Search with exact method'); ?></label></form>");
`

Reply:
Resolved with `.form-check-inline` which is a bootstrap utility class